cmd/serverless-init/initcontainer/initcontainer.go

```diff
@@ -61,30 +59,21 @@ func execute(logConfig *serverlessLog.Config, args []string) error {
 
 	cmd := exec.Command(commandName, commandArgs...)
 
-	shouldBuffer := calculateShouldBuffer(commandName)
```

Comment:
`calculateShouldBuffer` is now unused. Should we remove it? Or dotnet case is not yet addressed?

Reply:
Yes! Good catch